doc/dms_setup.md

```diff
@@ -0,0 +1,56 @@
+# Data Migration Service Setup
+
+## Overview
+
+There are a few parts to the setting up of DMS. The account running the DMS terraform needs to have certain AWS service access rights. Configure the VPC peering that is required for the DMS pipelining to other AWS accounts. Set the [create-cloudfoundry] pipeline against the appropriate environment. Make a note of any peering ids that are required. Setup the DMS configuration file named after the environment. Run the `create-cloudfoundry` job called `dms-terraform`.
+
+## Setup Steps
+
+1. Make sure that the concourse AWS account has enough rights to access DMS.
+1. There may be vpc peerings required. Ensure that they are setup before progressing further.
+1. Set the [create-cloudfoundry] pipeline against the appropriate environment. Make a note of the peering id(s) and the cidr block(s) associated with them.
+1. Secrets Manager has to hold the secrets for both the source and target databases. These secrets need to be accessible by the concourse AWS account for this terraform to work. The secret names must also be noted so that they can be used in the DMS configuration json file. The secrets file must contain the secrets shown in the example below:
+    ```yaml
+    {
+        "host": "string, holding the the DNS or the IP of the source/target RDS instance",
+        "password": "string, the master password of the RDS db",
+        "port": "integer, the port on which the RDS db is listening. Posibly on standard Postgresql pord 5432",
+        "username": "string, the admin username for the RDS db",
+        "database_name": "string, the db name to be migrated",
+        "engine_name": "string, in our case the value should be `postgres`"
+    }
+   ```
+2.  Within the terraform directory in this repo a ${env}.dms.json file is required to provide the DMS terraform with extra configuration. The detail required is shown in the example below:
```

Comment:
`${env}.dms.json` because we're referring to a file.

Reply:
👍